pom.xml

```diff
@@ -1471,6 +1474,100 @@
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>flatten-maven-plugin</artifactId>
+            <version>1.3.0</version>
+            <configuration>
+              <updatePomFile>true</updatePomFile>
+            </configuration>
+            <executions>
+              <execution>
+                <id>flatten</id>
+                <goals>
+                  <goal>flatten</goal>
+                </goals>
+                <phase>process-resources</phase>
+                <configuration>
+                  <keepCommentsInPom>true</keepCommentsInPom>
+                  <pomElements>
+                    <build>keep</build>
+                    <ciManagement>keep</ciManagement>
+                    <contributors>keep</contributors>
+                    <dependencies>keep</dependencies>
+                    <dependencyManagement>keep</dependencyManagement>
+                    <description>keep</description>
+                    <developers>keep</developers>
+                    <distributionManagement>keep</distributionManagement>
+                    <inceptionYear>keep</inceptionYear>
+                    <issueManagement>keep</issueManagement>
+                    <mailingLists>keep</mailingLists>
+                    <modules>keep</modules>
+                    <name>keep</name>
+                    <organization>keep</organization>
+                    <parent>keep</parent>
+                    <pluginManagement>keep</pluginManagement>
+                    <pluginRepositories>keep</pluginRepositories>
+                    <prerequisites>keep</prerequisites>
+                    <profiles>keep</profiles>
+                    <properties>keep</properties>
+                    <reporting>keep</reporting>
+                    <repositories>keep</repositories>
+                    <scm>interpolate</scm>
+                    <url>interpolate</url>
+                    <version>interpolate</version>
+                  </pomElements>
+                  <outputDirectory>${project.build.directory}</outputDirectory>
+                  <flattenedPomFilename>${project.artifactId}-${project.version}.pom</flattenedPomFilename>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <dependencies>
+              <dependency>
+                <groupId>io.jenkins.tools.incrementals</groupId>
+                <artifactId>incrementals-enforcer-rules</artifactId>
+                <version>1.4</version>
+              </dependency>
+            </dependencies>
+            <executions>
+              <execution>
+                <id>display-info</id>
+                <configuration>
+                  <rules>
+                    <rule implementation="io.jenkins.tools.incrementals.enforcer.RequireExtensionVersion">
+                      <version>[1.0-beta-4,)</version>
```

Comment:
Could probably be dropped since I guess we do not expect this profile to ever be used inherited, only locally.

Reply:
But that would increase the differential between this profile and the copypasta from which it originated, for no good reason I think. Better to keep all copies of this code as similar as possible I think.